src/csharp/CSharpParser.g4

```diff
@@ -104,10 +104,12 @@ non_assignment_expression
 	| query_expression
 	| conditional_expression
 	| throw_expression
+	// | unary_expression
```

Comment:
I would prefer not to see this in the history. Can you make sure your commits only contain non-debugging changes?